src/test/kotlin/biokotlin/featureTree/TypeSchemaTest.kt

```diff
@@ -304,7 +304,7 @@ class TypeSchemaTest : StringSpec({
             }
         }
 
-        File("src/test/resources/biokotlin/featureTree/dotOutput/type_schema_test.dot")
```

Comment:
Did you move this file?

Reply:
No, this file was generated.  It was throwing an error in the test because it did not make the directory before it wrote out the file.  I have it now just writing it the the featureTree/ directory which does exist.